python_modules/dagster/dagster/_core/storage/asset_check_execution_record.py

```diff
@@ -4,15 +4,27 @@
 from dagster import EventLogEntry
 
 
-class AssetCheckExecutionStatus(enum.Enum):
+# We store a limit set of statuses in the database, and then resolve them to the actual status
+# at read time. This is because the write path is to store a planned event (which creates a row
+# with PLANNED status) then update the row when we get the check result. But if the check never
+# runs, the row stays in PLANNED status.
+class AssetCheckExecutionRecordStatus(enum.Enum):
     PLANNED = "PLANNED"
     SUCCESS = "SUCCESS"
```

Comment:
Should this be SUCCEEDED to be consistent with FAILED? Also, same with EXECUTION_FAILURE below.